src/app/base/components/bricks-integrator/bricks-integrator.component.ts

```diff
@@ -55,6 +55,7 @@ export class BricksIntegratorComponent implements OnInit, OnDestroy {
     this.codeParser = new BricksCodeParser(this);
     const projectId = findProjectIdFromRoute(this.activatedRoute)
     this.dataRequestor = new BricksDataRequestor(this.projectApolloService, this.knowledgeBaseApollo, projectId, this);
+    this.functionType = this.router.url.indexOf("attributes") > -1 ? "Attribute" : this.router.url.indexOf("heuristics") > -1 ? "Heuristic" : "Function";
```

Comment:
i think the url is a bit unreliable here. especially since active learner (classes) and python functions have the same URL (besides a parameter). I'd go with an input parameter to be set for the component.